.github/actions/deploy/deploy.py

```
@@ -0,0 +1,1052 @@
+            f'until kubectl -n {self.deployment_namespace} get deployment {deployment_name} &> /dev/null; do echo "Waiting for the deployment {deployment_name}..."; sleep 10; done'
+        ]
+
+        result = self.run_command(wait_cmd, check=False)
+        if result.returncode != 0:
+            print(
+                f'❌ Deployment {deployment_name} was not created within 1 minute timeout'
+            )
+            self._investigate_deployment_failure(self.deployment_namespace,
+                                                 deployment_name)
+            raise RuntimeError(
+                f'Deployment {deployment_name} was not created within timeout')
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)